src/client/components/Fields.tsx

```diff
@@ -231,6 +287,23 @@ export function createLabel(text: string, required: boolean = true, className: s
   );
 }
 
+export const ApplicationInput: React.StatelessComponent<ApplicationInputProps> = ({
+  className = 'sd-form__input-text' ,
+  name,
+  type = 'text',
+  normalize,
+  placeholder,
+}) => (
```

Comment:
You could pull out the fields that don't match the "Field" prop types and then just use the spread operator for the rest. eg. `{errorTextInput, ...rest}` then `<Field component={errorTextInput} {...rest}/>`.

I think that's the syntax for the spread, I'm doing this without a linter so you'll have to let me know.